docs/source-pytorch/data/data.rst

```diff
@@ -0,0 +1,54 @@
+.. _data:
+
+Complex data uses
+=================
+
+.. raw:: html
+
+    <div class="display-card-container">
+        <div class="row">
+
+.. displayitem::
+   :header: LightningDataModules
+   :description: Introduction to the LightningDataModule
+   :col_css: col-md-4
+   :button_link: datamodule.html
+   :height: 150
+   :tag: basic
+
+.. displayitem::
+   :header: Iterables
+   :description: What is an iterable? How do I use them?
+   :col_css: col-md-4
+   :button_link: iterables.html
+   :height: 150
+   :tag: basic
+
+.. displayitem::
+   :header: Access your data
+   :description: How to access your dataloaders
+   :col_css: col-md-4
+   :button_link: access.html
+   :height: 150
+   :tag: basic
+
+.. displayitem::
+   :header: Infinte or streaming datasets
+   :description: Using iterable-style datasets with Lightning
+   :col_css: col-md-4
+   :button_link: infinite.html
+   :height: 150
+   :tag: basic
+
+.. displayitem::
+   :header: Faster DataLoaders
+   :description: How alternative dataloader projects can be used with Lightning
+   :col_css: col-md-4
+   :button_link: alternatives.html
+   :height: 150
+   :tag: basic
```

Comment:
I would maybe promote these two to the "intermediate" or "advanced" difficulty level.